Sources/Shared/Models/VIMVideoProgressiveFile.h

```diff
@@ -25,7 +25,12 @@
 //
 
 @import Foundation;
+
+#if TARGET_OS_IPHONE || TARGET_OS_IOS
```

Comment:
Are we actually using UIKit / AppKit for anything in this file?  If we're not doing any UI related work, I'd rather get rid of the unused headers.

Reply:
There's a `CGSize` there that is defined in UIKit :/